tests/test_manager.py

```diff
@@ -159,6 +159,13 @@ async def http_no_assert(
     )
     # ...
 
+async def returns_without_handling_lifespan(
+    scope: dict, receive: typing.Callable, send: typing.Callable
+) -> None:
+    if scope["type"] != "http":
+        ...
+    raise AssertionError("Should not be called")  # pragma: no cover
```

Comment:
I believe we need to `return` here. Otherwise the app continues to raise the `AssertionError`. This makes it fail and go through the `except BaseException` branch, instead of going through the new `if not receive_called` branch, which explains the "Line 66 not covered" error in CI.
```suggestion
        return
    raise AssertionError("Should not be called")  # pragma: no cover
```